src/pages/index.astro

```diff
@@ -5,10 +5,12 @@ import MentorsComponent from '@components/mentors.component';
 import BlogComponent from '@components/blog/blog.component';
 import Layout from '@layout/Layout.astro';
 import { FooterComponent } from '@components/footer.component';
+import NavbarComponent from '@components/navbar.component';
 ---
 
 <Layout>
   <main>
+    <NavbarComponent client:visible />
```

Comment:
This should be part of the layout component, otherwise we will need to add it manually to every page

Reply:
agree